src/gwt/src/org/rstudio/studio/client/projects/ui/prefs/ProjectGeneralPreferencesPane.java

```diff
@@ -56,8 +56,11 @@ public ProjectGeneralPreferencesPane(Session session)
       grid.setWidget(3, 0, new Label("Always save history (even if not saving .RData)"));
       grid.setWidget(3, 1, alwaysSaveHistory_ = new YesNoAskDefault(false));
 
+      // execute .Rprofile
+      grid.setWidget(4, 0, executeRprofile_ = new CheckBox("Execute .Rprofile on session start"));
```

Comment:
Do we know if we're in packrat mode? If we do, can we disable or hide this in packrat mode (since it doesn't actually do anything there)?